hyperspy/io_plugins/bcf.py

```diff
@@ -1029,7 +1104,7 @@ def bcf_imagery(obj_bcf, instrument=None):
                      # possibilities to get such parameter from bruker
                      # or some SEM's'
                      'beam_energy': obj_bcf.header.sem.hv,
-                     'tilt_stage': obj_bcf.header.stage.tilt_angle,
+                     'tilt_stage': obj_bcf.header.stage.tilt_stage,
                      'stage_x': obj_bcf.header.stage.x,
                      'stage_y': obj_bcf.header.stage.y
```

Comment:
See here to know how  [mapping](https://github.com/hyperspy/hyperspy/blob/a9ce12db16d33952c05ec3e7b3d4de41f10af9d2/hyperspy/io.py#L391) works. Actually, in this case you may need to add more data to original_metadata, because it worked nicely only when you parse one parameter from the original_metadata dictionary! The [blockfile reader](https://github.com/hyperspy/hyperspy/blob/e130e34f347f236bfaf136add8a1629768468a4c/hyperspy/io_plugins/blockfile.py#L63) is simple and good example. In the dm plugin, there are a [couple of functions](https://github.com/ericpre/hyperspy/blob/f96a8e6c7d7aba43e254c598b488f5fab28a9d01/hyperspy/io_plugins/digital_micrograph.py#L795) to parse metadata that returns `None` and in this case the metadata tree isn't populated with entries.

Please do as you prefer and if you can avoid populating metadata with `None` value, it would be great! :) When testing on my files, I got a `None` for `stage_x`, `stage_y` and `stage_tilt`.